composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "dfmedia/wp-api-co-author-plus-endpoints",
+    "description": "This plugin utilizes WP-API to query for Co-Authors Plus data and add authors to a post via the author term.",
+    "homepage": "https://github.com/jjgrainger/wp-custom-post-type-class",
```

Comment:
@misfist this should be changed to: `https://github.com/dfmedia/wp-api-co-author-plus-endpoints`